src/ConstructiveSolidGeometry/Transformations.jl

```diff
@@ -14,7 +14,7 @@ struct RotatedGeometry{T,P<:AbstractGeometry{T},RT} <: AbstractGeometry{T}
 end
 in(p::CartesianPoint, g::RotatedGeometry) = in(g.inv_r * p, g.p)
 rotate(g::AbstractGeometry{T}, r::RotMatrix{3,RT,9}) where {T,RT} = RotatedGeometry(g, r)
-rotate(g::RotatedGeometry{T,<:Any,RT}, r::RotMatrix{3,RT,9}) where {T,RT} = RotatedGeometry{T,typeof(g),RT}(g.p, inv(r) * gp.r)
+rotate(g::RotatedGeometry{T,<:Any,RT}, r::RotMatrix{3,RT,9}) where {T,RT} = RotatedGeometry(g.p, r * inv(g.inv_r))
```

Comment:
I believe you just added one inversion. Now `rotate` will call `inv` two times.
Or is this beneficial rounding/numerical-precesion wise?
Same for `scale(g::ScaledGeometry, s::SVector{3,T})`.

Reply:
This gave an error when I called it. I just realized this was due to `gp.r` and not `g.r` in the arguments.
I can change it back

Reply:
Hmm... I just recognized the following:
```julia
using Rotations, StaticArrays
m1 = RotMatrix(RotX(67) * RotY(45) * RotZ(13))
m2 = RotMatrix(RotX(-1.67) * RotY(5) * RotZ(.45))
m1i = inv(m1)
m2i = inv(m2)
m1i * m2i ≈ inv(m2 * m1) # true
```
So we have to change the order to : 
```julia
rotate(g::RotatedGeometry{T,<:Any,RT}, r::RotMatrix{3,RT,9}) where {T,RT} = RotatedGeometry{T,typeof(g),RT}(g.p,gp.r * inv(r))`
```
Am i right?

However, there are some rounding issues
```julia
m1i * m2i == inv(m2 * m1) # false
```
So I would go for actually inverting it back to `m1` times as I believe it is numerically better. 
But, I am not sure here...

Reply:
Apparently, calling `RotatedGeometry{...}` and `ScaledGeometry{...}` is not possible and results in errors.
I added another function into the struct that will handle repeated transformations accordingly.
The results are tested using the repeated function calls in `CSG_test.jl`
Note the order of the matrices in `RotatedGeometry` that is correct now.

Reply:
Because `inv(m1i) == m1` and `inv(m2i) == m2`

Reply:
I just reverted my commit, now it is inverting it back.